src/components/DayPicker.jsx

```diff
@@ -395,8 +404,12 @@ export default class DayPicker extends React.Component {
       this.props.onNextMonthClick(e);
     }
 
-    const translationValue =
-      this.isVertical() ? -this.getMonthHeightByIndex(1) : -this.dayPickerWidth;
+    const convertedTranslationValueRTL = (this.dayPickerWidth + this.state.translationValueRTL);
+    let translationValue = this.props.isRTL ? convertedTranslationValueRTL : -this.dayPickerWidth;
+
+    if (this.isVertical()) {
+      translationValue = this.getMonthHeightByIndex(1) * -1;
```

Comment:
`translationValue = -this.getMonthHeightByIndex(1);`

also, why the hardcoded 1 here?